src/pages/axa-ch-main/story.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { storiesOf } from '@storybook/html';
+import { html, render } from 'lit-html';
+import { boolean, withKnobs } from '@storybook/addon-knobs';
+
+import Readme from '../../../README.md';
+import '../../components/10-atoms/text';
+import '../../components/10-atoms/heading';
+import commercialHeroBanner from './commercial-hero-banner';
+import footer from './footer';
+import policyFeatures from './policy-features';
+import navbar from './navbar';
+import footerSmall from './footer-small';
```

Comment:
You are wrapping our components to have a cleaner story.js?

Reply:
Yes, it help's with readabiliy and makes it reusable.